OG_Format.adoc

```diff
@@ -722,7 +722,7 @@ PARAMETER_UNITS:parameter_units_vocabulary = “”;
 = Geophysical variables
 ////
 = Geophysical variables
-
+"We recommend using NaN for float/double, but the user is allowed to choose any value with the same data type of the variable."
```

Comment:
Maybe add a reminder that this value should be out of that sensor's range of possible measurements? I know it should be obvious, but it won't hurt to be explicit.

I'm OK if you think it is too much.